api/v1alpha1/nodepool_types.go

```diff
@@ -701,6 +701,16 @@ type Volume struct {
 	//
 	// +optional
 	IOPS int64 `json:"iops,omitempty"`
+
+	// Encrypted is whether the volume should be encrypted or not.
+	// +optional
```

Comment:
can we please make this immutable?
`// +kubebuilder:validation:XValidation:rule="self == oldSelf", message="Encrypted is immutable"`